app/controllers/messages_controller.rb

```diff
@@ -8,9 +8,16 @@ class MessagesController < ApplicationController
   def create
     @message = @conversation.messages.create(message_params)
     EmojiFiller.call(@message)
-    respond_to do |format|
-      format.html
-      format.js
+    if @message.valid?
+      respond_to do |format|
+        format.html
+        format.js
+      end
+    else
+      @message.errors.full_messages.each do |error_message|
+        flash[:error] = error_message
+      end
```

Comment:
You could write it in just one line: 
```suggestion
      @message.errors.full_messages.each  { |error_message| flash[:error] = error_message }
```

Reply:
Correct :+1: